app/components/avo/fields/text_field/edit_component.html.erb

```diff
@@ -7,6 +7,7 @@
     placeholder: @field.placeholder,
     style: @field.get_html(:style, view: view, element: :input),
     multiple: multiple,
-    autocomplete: @field.autocomplete
+    autocomplete: @field.autocomplete,
+    autofocus: index.zero?
```

Comment:
`autofocus` should be passed when initializing the field and not computed here.

For this feature we want to add focus when first field on an action.

```suggestion
    autofocus: @field.autofocus?
```

And lets add `autofocus` as an option for all components of fields types that can be focused.